target/linux/bcm53xx/base-files/etc/board.d/02_network

```diff
@@ -31,6 +31,10 @@ bcm53xx_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan" "1:lan" "2:lan" "3:lan" "4:wan" "5t@eth0"
 		;;
+	linksys,panamera)
+		ucidef_add_switch "switch1" \
+			"0:lan" "1:lan:7" "2:lan:4" "3:lan:8" "4:wan" "5t@eth0"
```

Comment:
What's the :7, :4; :8 supposed to achieve? Are these number actually used in Luci?

Reply:
I assumed these labeled the physical ports on the device, but I didn't notice them in UI. I can get rid of them.

Reply:
x:lan:y
The left number x is the internal port number of the switch, and the right number y is something corresponding to the label, if that deviates from internal order. However, I was under the impression that LuCI would not allow gaps between these numbers, but use consecutive numbering anyway ...

Reply:
For now I'll just remove the ys. Thanks

Reply:
Done

Reply:
@adschm: that `x:lan:y` syntax works for me and I find it very useful. It results in adding `index` to switch ports in the `/etc/board.json`. UI uses that to display LAN port names, see:
![switch](https://user-images.githubusercontent.com/452972/113254437-dd5d6f00-92c6-11eb-88ef-4911cf1e60fa.png)

Reply:
Yes, my main point was that I don't understand why we need 7 and 8 here. Normally, the numbers for Luci would be :1 to :4 if there is four lan ports. But if it works now, it's probably not worth touching it again.

Reply:
This device has 8 LAN ports connected to 2 switches

Reply:
But only four are shown and numbered 1-4 in Luci, so it's technically not required. Anyway, since this preserves additional information, we might as well keep it :-)